<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New</class>
 <widget class="QDialog" name="New">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建菜品</string>
  </property>
  <layout class="QVBoxLayout" name="vlay" stretch="1,0,1">
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label1">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>新建菜品</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="flay">
     <property name="horizontalSpacing">
      <number>36</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledit1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>价格</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledit2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlay2">
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="pbtn1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="pbtn2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="pbtn3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>创建</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
